app/backend/approaches/chatapproach.py

```diff
@@ -47,15 +34,17 @@ async def run_until_final_call(self, messages, overrides, auth_claims, should_st
 
     def get_system_prompt(self, override_prompt: Optional[str], follow_up_questions_prompt: str) -> str:
         if override_prompt is None:
-            return self.system_message_chat_conversation.format(
-                injected_prompt="", follow_up_questions_prompt=follow_up_questions_prompt
+            return self.system_message_chat_conversation_template.render(
+                follow_up_questions_prompt=follow_up_questions_prompt,
+                injected_prompt=""
             )
         elif override_prompt.startswith(">>>"):
-            return self.system_message_chat_conversation.format(
-                injected_prompt=override_prompt[3:] + "\n", follow_up_questions_prompt=follow_up_questions_prompt
+            return self.system_message_chat_conversation_template.render(
+                follow_up_questions_prompt=follow_up_questions_prompt,
+                injected_prompt=override_prompt[3:] + "\n"
             )
         else:
-            return override_prompt.format(follow_up_questions_prompt=follow_up_questions_prompt)
+            return override_prompt.format(follow_up_questions_prompt=follow_up_questions_prompt) ## check if correct
```

Comment:
Is "check if correct" a TODO or yourself?

I think this would still be correct, as override_prompt would just be a string, not a jinja template.

Reply:
I forgot to remove my TODO comment. Thanks for catching that!